autoptsserver.py

```diff
@@ -284,7 +286,9 @@ def server_init(self):
 
         if self._args.ykush:
             power_dongle(self._args.ykush, False)
+            sleep(1)
```

Comment:
are those sleeps really needed?

Reply:
Fixed. Out of curiosity I measured how long it takes for the PTS dongle to become visible. In the case of newer ones, seen as COM ports, it takes around 590ms. The older ones, seen as USB, need around 285ms.